### ISSUE TITLE
great game!

### ISSUE THREAD

**Issue by adbenitez** [rottencandy/js13k2024#1]
hi, just wanted to say this is one of my favorite entries to the js13k 2024 jam, really nice and addictive game (I have won it several times already and can't get enough of it :sweat_smile: ) I hope the project keeps alive beyond the jam.

Something I could recommend is some balancing to the shield, it gets to a point where I don't need to move anymore and can basically stay still and the monsters can't reach me. Also if the game could be longer or infinite (with increasing difficulty) that would be awesome!

btw, I [ported](https://github.com/DeltaZen/xiicur-surviivors) the game to [Webxdc](https://webxdc.org/) (sandboxed mini-games shared in a chat/group with possibility to sync score boards and/or "Player Foo scored X" in-chat messages between chat members)

**Comment by rottencandy** [comment]
Thank you!

Yes I've been trying to think of ways to balance out the mechanics. It's quite hard to make sure the difficulty curve stays linear. I've also been thinking of somehow discouraging picking up some of the easier powerups by making the harder ones more rewarding to use.

My original plan was to let players go on infinitely until they're defeated. But I realized it gets boring and repetitive quickly because there's no new content after a while, and it wasn't as satisfying because you can never really beat the game that way.

Never heard of webxdc before, but looks interesting. Trying to learn more about it, but it seems like an open project and not profiting off of the game so okay by me.

**Comment by adbenitez** [comment]
> My original plan was to let players go on infinitely until they're defeated. But I realized it gets boring and repetitive quickly because there's no new content after a while, and it wasn't as satisfying because you can never really beat the game that way.

good point, maybe what could be done to add more fun difficulty is to have a boss per wave "season" ex. basically the same slime sprite but bigger with more damage and HP, etc. (and maybe resistance to some of the weapon types?) that could get the player out of the "comfort zone"
